### PR TITLE
A/B homepage (Added Animations to the icons)

### PR DESCRIPTION
**Pull request by DannyRuchtie**
- Added animations to the icons to invoke interaction
- Added title tag for accessibility

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3996/commits/644d1dbe10539c6dbd0b2a902bfabde220c101c4 failed.

**Comment by DannyRuchtie** [comment]
@garethx There was a merge conflict in index and index2.
I think I solved it correctly, could you please double check?

File: packages/homepage/src/pages/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { identify } from '@codesandbox/common/lib/utils/analytics';
```
Comment (MichaelDeBoey) [inline packages/homepage/src/pages/index.js:2]: This one's imported, but not used

```suggestion
```

**Comment by MichaelDeBoey** [comment]
@DannyRuchtie Maybe also run `Prettier` on the changed files? 🤔

**Pull request merged**